nvflare/edge/assessors/count.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+
+from nvflare.apis.fl_context import FLContext
+from nvflare.app_common.abstract.aggregator import Aggregator
+from nvflare.edge.assessor import Assessment, Assessor
+
+
+class CountAssessor(Assessor):
+    def __init__(self, min_count: int, max_count: int, timeout: float):
+        Assessor.__init__(self)
+        self.min_count = min_count
+        self.max_count = max_count
+        self.timeout = timeout
+        self._aggregator = None
+        self._start_time = None
+
+    def initialize(self, aggregator: Aggregator, fl_ctx: FLContext):
+        self._aggregator = aggregator
+        if not hasattr(aggregator, "get_count"):
```

Comment:
Should we define a new aggregator with get_count method? This is duck-taping and a more flexible way to enforce type.

Reply:
I thought about this but then decided against it, because this could potentially set a bad example for defining all kinds of special aggregators since each assessor may require different types of methods from the aggregator.

Duck-typing is a good feature of Python, why not make use of it.